content/zh/blog/2019/egress-traffic-control-in-istio-part-1/index.md

```diff
@@ -39,13 +39,13 @@ _1.3.4 不允许持卡人数据环境没有授权的出站流量进入互联网
 
 出口流量安全管控的意思是监控出口流量并且针对出口流量应用所有的安全策略。
 监控出口流量，可以对它进行分析（可能是离线的），即便你无法实时阻止攻击，也要检测攻击事件。
-另外一个减少攻击可能性的方法是遵循[需要知道](https://en.wikipedia.org/wiki/Need_to_know#In_computer_technology])的原则进行指定限制访问策略。
+另外一个减少攻击可能性的方法是遵循 [需要知道](https://en.wikipedia.org/wiki/Need_to_know#In_computer_technology]) 的原则进行指定限制访问策略。
```

Comment:
```suggestion
另外一个减少攻击可能性的方法是遵循[需要知道](https://en.wikipedia.org/wiki/Need_to_know#In_computer_technology])的原则进行指定限制访问策略。
```
This kind of space is unnecessary because it will be `遵循需要知道的原则` after build. All are Chinese. Same issues below.